magento/pwa-studio
# [PWA-451] Fix buildpack test snaps

## Description

Issue - Unable to run release scripts as snapshot is hardcoded to 5.0.0.
Expected - User pattern match to overcome this.

## Related Issue





* [[PWA-451](https://jira.corp.magento.com/browse/PWA-451)] Fix buildpack test snaps

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Manually change version to `5.1.0-alpha.1` in `packages/pwa-buildpack/package.json`
2. Run tests - verify no failures 🎉 

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
